### ISSUE TITLE
[BUG] cudf nightly throws std::logic_error on import in jupyterlab

### ISSUE THREAD

**Issue by randerzander** [rapidsai/cudf#3747]
With cudatoolkit=10.1, using several of the latest nightly conda packages:
```
(RAPIDS) rgelhausen@randy-m:~$ conda list | grep rapids
cudf                      0.12.0b200109         py37_1045    rapidsai-nightly
dask-cuda                 0.12.0a200109           py37_47    rapidsai-nightly
dask-cudf                 0.12.0b200109         py37_1045    rapidsai-nightly
libcudf                   0.12.0b200109     cuda10.1_1024    rapidsai-nightly
libnvstrings              0.12.0b200109     cuda10.1_1024    rapidsai-nightly
librmm                    0.12.0a200109      cuda10.1_155    rapidsai-nightly
nvstrings                 0.12.0b200109         py37_1045    rapidsai-nightly
rmm                       0.12.0a200109          py37_155    rapidsai-nightly
```

```
(RAPIDS) rgelhausen@randy-m:~$ cat /usr/local/share/jupyter/kernels/python3/kernel.json 
{
 "argv": [
  "/opt/conda/anaconda/envs/RAPIDS/bin/python",
  "-m",
  "ipykernel_launcher",
  "-f",
  "{connection_file}"
 ],
 "display_name": "Python 3",
 "language": "python"
}
```

Starting the specific python interpreter directly and importing cudf is fine:
```
(RAPIDS) rgelhausen@randy-m:~$ /opt/conda/anaconda/envs/RAPIDS/bin/python
Python 3.7.6 | packaged by conda-forge | (default, Jan  7 2020, 22:33:48) 
[GCC 7.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import cudf
>>> 
```

But when using the same kernel, importing cudf from inside a Jupyterlab notebook or console fails. Jupyterlab logs:
```
[D 22:24:02.102 NotebookApp] activity on f18c1a44-0a42-4299-a454-4df3ad358513: status
[D 22:24:02.104 NotebookApp] activity on f18c1a44-0a42-4299-a454-4df3ad358513: execute_input
terminate called after throwing an instance of 'std::logic_error'
  what():  basic_string::_M_construct null not valid
[I 22:24:03.981 NotebookApp] KernelRestarter: restarting kernel (1/5), keep random ports
kernel f18c1a44-0a42-4299-a454-4df3ad358513 restarted
[D 22:24:03.982 NotebookApp] Starting kernel: [u'/opt/conda/anaconda/envs/RAPIDS/bin/python', u'-m', u'ipykernel_launcher', u'-f', u'/root/.local/share/jupyter/runtime/kernel-f18c1a44-0a42-4299-a454-4df3ad358513.json']
[D 22:24:03.987 NotebookApp] Connecting to: tcp://127.0.0.1:50671
[D 22:24:06.988 NotebookApp] KernelRestarter: restart apparently succeeded
```

**Comment by jrhemstad** [comment]
This looks to be the same as https://github.com/rapidsai/cudf/issues/3722

**Comment by beckernick** [comment]
cc @Nanthini10 is that what you were experiencing?

**Comment by Nanthini10** [comment]
No, this was not the error.

**Comment by randerzander** [comment]
This is resolved for me after #3699
